migration-roadmap/infield-button.md

```diff
@@ -0,0 +1,414 @@
+    </div>
+</button>
+```
+
+</details>
+
+<details>
+<summary>Diff: Legacy (CSS main) → Spectrum 2 (CSS spectrum-two)</summary>
+
+### HTML Output Comparison
+
+**Legacy (CSS main branch) - Single button:**
+
+```html
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-add class="spectrum-InfieldButton-icon"></sp-icon-add>
+    </div>
+</button>
+```
+
+**Legacy (CSS main branch) - Stacked buttons:**
+
+```html
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM spectrum-InfieldButton--top"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+    aria-label="add"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-chevronup75
+            class="spectrum-InfieldButton-icon"
+        ></sp-icon-chevronup75>
+    </div>
+</button>
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM spectrum-InfieldButton--bottom"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+    aria-label="add"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-chevrondown75
+            class="spectrum-InfieldButton-icon"
+        ></sp-icon-chevrondown75>
+    </div>
+</button>
+```
+
+**Spectrum 2 (CSS spectrum-two branch) - Single button:**
+
+```html
+<button
+    class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+    aria-haspopup="listbox"
+    type="button"
+    tabindex="0"
+    role="presentation"
+>
+    <div class="spectrum-InfieldButton-fill">
+        <sp-icon-add100 class="spectrum-InfieldButton-icon"></sp-icon-add100>
+    </div>
+</button>
+```
+
+**Spectrum 2 (CSS spectrum-two branch) - Inline buttons:**
+
+```html
+<div class="spectrum-InfieldButton-inline">
+    <button
+        class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+        aria-haspopup="listbox"
+        type="button"
+        tabindex="0"
+        aria-label="minus"
+    >
+        <div class="spectrum-InfieldButton-fill">
+            <sp-icon-dash class="spectrum-InfieldButton-icon"></sp-icon-dash>
+        </div>
+    </button>
+    <button
+        class="spectrum-InfieldButton spectrum-InfieldButton--sizeM"
+        aria-haspopup="listbox"
+        type="button"
+        tabindex="0"
+        aria-label="add"
+    >
+        <div class="spectrum-InfieldButton-fill">
+            <sp-icon-add class="spectrum-InfieldButton-icon"></sp-icon-add>
+        </div>
+    </button>
+</div>
+```
+
+### Key Changes in HTML Structure
+
+1. **Layout paradigm shift**: Replaced `isStacked` vertical stacking with `isInline` horizontal grouping
+2. **Enhanced iconography**:
+    - Single button: Uses size-specific icon names (e.g., `Add100` instead of `Add`)
+    - Inline buttons: Switched from chevron icons to more intuitive action icons (`Dash`/`Add`)
```

Comment:
I don't know if "action icons" is really how we would describe these 😆 They're just workflow icons instead now.